packages/core/getUserAvailability.ts

```diff
@@ -339,10 +339,11 @@ const _getUserAvailability = async function getUsersWorkingHoursLifeTheUniverseA
   });
 
   const formattedBusyTimes = detailedBusyTimes.map((busy) => ({
-    start: dayjs(busy.start),
-    end: dayjs(busy.end),
+    start: dayjs(busy.start).tz(timeZone),
+    end: dayjs(busy.end).tz(timeZone),
   }));
 
+  //INFO  formattedBusyTimes need to be in the same tz as dateRanges to subtract correctly.
```

Comment:
`Dayjs().isBefore()/isAfter()` is not affected by the tz. But while subtracting, it will push the `Dayjs` object (which can be in diff tz) in the returned value(array).

And the fn (ex. getSlots) who is dependant on the returned value can get affected by it & produce incorrect behaivour.

Reply:
Was that a bug? if yes, how can I reproduce that bug?